content/source/docs/plugin/framework/publishing.html.md

```diff
@@ -8,35 +8,26 @@ description: |-
 
 # Publishing to the Registry
 
-Publishing providers built on the provider development framework to the
-Terraform Registry works the same as [publishing providers built on
-SDKv2](/docs/registry/providers/publishing.html), with a minor difference.
-Providers built on the framework are built on version 6 of the Terraform
-protocol. The Registry, by default, assumes providers being published support
-version 5 of the Terraform protocol, which providers built on the framework do
-not. The Registry needs to be told about this difference in protocol version
-support. This is done by adding a manifest to the provider's release assets.
+You can publish providers to the Terraform Registry using the same [process as providers built on SDKv2](/docs/registry/providers/publishing.html), with one extra step.
+
+The Terraform Registry assumes that published providers support
+version 5 of the Terraform protocol, but providers built using the framework are built on version 6. You need to add a manifest to your provider's release assets that tells the Terraform Registry about this difference.
 
 ## Add a Version Manifest
 
-When uploading your new provider version to GitHub, next to the zip files
-containing your binaries for each platform, include a file named
-`terraform-provider-$NAME_$VERSION_manifest.json` (where `$NAME` is your
-provider's name, like `random` and `$VERSION` is your provider's version, like
-`1.2.3`) with the following contents:
+When you upload your new provider version to GitHub:
+
+1. Next to the zip files containing your binaries for each platform, include a file named `terraform-provider-$NAME_$VERSION_manifest.json` (where `$NAME` is your provider's name, like `random` and `$VERSION` is your provider's version, like `1.2.3`). Add the following contents:
 
-```
+    ```
 {
   "version": 1,
   "metadata": {
     "protocol_versions": ["6.0"],
   },
 }
-```
+    ```
 
-Include the SHA-256 checksum of this JSON file in your `SHA256SUMS` file.
+2. Include the SHA-256 checksum of this JSON file in your `SHA256SUMS` file.
 
-The registry will detect this file and understand this version of your provider
-only supports version 6 of the Terraform protocol, and will correctly advertise
-that fact to Terraform, so Terraform versions that don't support protocol 6
-will not download it.
+The Terraform Registry will detect this file and understand that this version of your provider only supports version 6 of the Terraform protocol. It and will correctly advertise that fact to Terraform, so that Terraform versions that don't support protocol 6 will not download it.
```

Comment:
```suggestion
The Terraform Registry will detect this file and understand that this version of your provider only supports version 6 of the Terraform protocol. It will correctly advertise that fact to Terraform, so that Terraform versions that don't support protocol 6 will not download it.
```